moveit_core/robot_state/include/moveit/robot_state/cartesian_interpolator.h

```diff
@@ -44,6 +44,14 @@ namespace moveit
 {
 namespace core
 {
+/** Struct defining linear and rotational precision */
+struct CartesianPrecision
+{
+  double translational = 0.001;
+  double rotational = 0.01;
+  double max_resolution = 1e-5;
```

Comment:
Can you add brief docstrings to each of these, which include units?

I can make sense of `translational` and `rotational`, but without reading the rest of the code I don't know what `max_resolution` means.